.github/workflows/tests.yml

```diff
@@ -8,7 +8,13 @@ on:
 
 jobs:
   test:
-    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        os: ['ubuntu-20.04']
+        ruby-version: ['3.0', '3.1']
```

Comment:
I've added a matrix for testing. Let me know which values you want to test with.